### PR TITLE
Resolved many typos in comments, Exceptions and outputs

### PR DESCRIPTION
**Pull request by tomaarsen**
Hello!

### Pull request overview
* Resolved many typos in comments, Exceptions and outputs, using the `codespell` module.

---

### Note
Despite the `codespell` module working quite well, it cannot properly be automated (i.e. in per-commit or GitHub Actions CI), as it attempts to "fix" non-English text, URLs, tests and more.

- Tom Aarsen

### CODE REVIEW

File: nltk/app/chartparser_app.py [hunk 1]
```diff
@@ -746,7 +746,7 @@ def _init_buttons(self, root):
         Frame(buttons, width=20).pack(side="left")
         Button(buttons, text="Swap Charts", command=self._swapcharts).pack(side="left")
 
-        Button(buttons, text="Detatch Output", command=self._detatch_out).pack(
+        Button(buttons, text="Detach Output", command=self._detatch_out).pack(
```
Comment (purificant) [inline nltk/app/chartparser_app.py]: Same typo is also in the method name `_detatch_out`

File: nltk/test/twitter.ipynb [hunk 2]
```diff
@@ -833,103 +834,146 @@
...
-    "    json2csv_entities(fp, 'tweets.20150430-223406.place.csv',\n",
-    "                        ['id', 'text'], 'place', ['name', 'country'])\n",
-    "\n",
-    "with open(input_file) as fp:\n",
-    "    json2csv_entities(fp, 'tweets.20150430-223406.place_bounding_box.csv',\n",
+    "from nltk.twitter.common import json2csv_entities\r\n",
+    "with open(input_file) as fp:\r\n",
+    "    json2csv_entities(fp, 'tweets.20150430-223406.hashtags.csv',\r\n",
+    "                        ['id', 'text'], 'hashtags', ['text'])\r\n",
+    "    \r\n",
```
Comment (purificant) [inline nltk/test/twitter.ipynb]: Was it intended to include all these changes to the `twitter.ipynb` file?

**Comment by purificant** [comment]
Love this! :+1: :heart:

File: nltk/app/chartparser_app.py (hunk 1 above)
Comment (tomaarsen) [reply nltk/app/chartparser_app.py]: Wonderful, thank you! I'm not sure why `codespell` didn't detect it. It did try to update other instances of misspelled functions and methods.

**Review by purificant — CHANGES_REQUESTED** [review]

File: nltk/test/twitter.ipynb (hunk 2 above)
Comment (tomaarsen) [reply nltk/test/twitter.ipynb]: It was not - this must be a consequence of my IDE reformatting the file automatically upon saving, how annoying. There was just one typo in this file. It seems that the only changes are line endings and the order of dictionary/object keys, so it's not a big deal, but I'll revert this. Thank you for pointing this out.

Comment (tomaarsen) [reply nltk/test/twitter.ipynb]: I'll have a look at it tomorrow. I tried to use the same git on a different OS, and it's not a big fan of that.

**Review by purificant — APPROVED** [review]

**Comment by stevenbird** [comment]
Wow... this is great

**Pull request merged**